tests/robustness/makefile.mk

```diff
@@ -3,11 +3,14 @@ test-robustness-reports: export GOTOOLCHAIN := go$(shell cat .go-version)
 test-robustness-reports:
 	cd ./tests && go test ./robustness/validate -v --count 1 --run TestDataReports
 
+test-robustness-release-3.6: /tmp/etcd-release-3.5-failpoints/bin
```

Comment:
Targets `/tmp/etcd-release-3.5-failpoints/bin` don't get cashed well. Will need to fix that if you want to use it.